Howard-Heaven-Submission/README.md

```diff
@@ -0,0 +1 @@
+In this code I decided to go with a Ceasar Cypher algorithm, where you can give it a word to decrypt or encrypt by giving it a number of how much to increment by. For instance if type "hello" as my word to encrypt and enter the number 4 for example it will encrypt the word to be "lipps" and if i repeat the same process backwards it will provide "hello" instead.
```

Comment:
I would suggest including a bit more in this readme, such as how to run the python file, and how the directories are organized.